frontend/packages/knative-plugin/src/components/functions/GettingStartedSection.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Card, CardBody, ExpandableSection } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { FunctionsDocsGettingStartedCard } from './FunctionsDocsGettingStartedCard';
+import { QuickStartGettingStartedCard } from './QuickStartGettingStartedCard';
+import { SampleGettingStartedCard } from './SamplesGettingStartedCard';
+
+import './GettingStartedSection.scss';
+
+export const GettingStartedSection: React.FC = () => {
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState(true);
+
+  return (
+    <div className="odc-functions-getting-started-section">
+      <ExpandableSection
+        toggleText={t('knative-plugin~Get started with functions')}
+        onToggle={() => setIsOpen(!isOpen)}
+        isExpanded={isOpen}
+        displaySize="large"
+      >
+        <Card className="odc-functions-getting-started-grid" data-test="getting-started">
+          <span style={{ marginLeft: '16px' }}>
+            {t('knative-plugin~choose how to create a function from 3 methods')}
+          </span>
```

Comment:
I don't think that's correct. The quick start will show "different ways". I recommend removing this sentence. Please check this with Naina and Ethan.

```suggestion
```